pkg/services/navtree/navtreeimpl/admin.go

```diff
@@ -136,6 +136,17 @@ func (s *ServiceImpl) getServerAdminNode(c *contextmodel.ReqContext) *navtree.Na
 		}
 	}
 
+	authConfigUIAvailable := s.license.FeatureEnabled("saml") && !s.features.IsEnabled(featuremgmt.FlagDisableAuthenticationConfigUI)
```

Comment:
Is it necessary to check if saml is enabled here especially if we reverse the feature flag?

Reply:
Yeah, but I guess we remove this check when we start adding support for other providers?

Reply:
Yes, once we add more providers, we remove this check.